purldb-toolkit/README.rst

```diff
@@ -1,12 +1,17 @@
 purldb-toolkit
 ==============
 
+.. contents:: :local:
+    :depth: 3
+
```

Comment:
One blank line is enough here, instead of 3.

Reply:
Done.